pkg/helpers/homebrew.go

```diff
@@ -0,0 +1,60 @@
+package helpers
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/pior/dad/pkg/env"
+	"github.com/pior/dad/pkg/utils"
+)
+
+type Homebrew struct {
+	prefix string
+}
+
+// NewHomebrew is returning a new Cellar
+func NewHomebrew() *Homebrew {
+	return NewHomebrewWithPrefix("/usr/local")
+}
+
+func NewHomebrewWithPrefix(prefix string) *Homebrew {
+	if !utils.PathExists(filepath.Join(prefix, "Cellar")) {
+
+		path := strings.Split(env.NewFromOS().Get("PATH"), ":")
```

Comment:
We could probably update the Env api to make `getAndSplitPath()` public.
Renaming the 2 symmetric methods could be a good idea: `GetPathParts()`/`SetPathParts()`